amads/algorithms/scale.py

```diff
@@ -27,17 +27,17 @@
 def scale(score, factor=2, dim="all"):
     if dim == "all":
         scale(score, factor, "duration")
-        scale(score, factor, "start")
+        scale(score, factor, "onset")
         return score
     for elem in score.content:
         if isinstance(elem, EventGroup):
             scale(elem, factor, dim)
-            if dim == "start":
-                elem.start *= 2
+            if dim == "onset":
+                elem.onset *= 2
```

Comment:
I thought I saw somewhere this was changed to elem.onset *= factor (and it should be)

Reply:
Yes, you're right. I'll change it here too just to be safe.